core/storage/suggestion/gae_models.py

```diff
@@ -1990,3 +2000,1275 @@ def export_data(
+                )).order(sort)
+        else:
+            sort_query = cls.query(
+                datastore_services.all_of(
+                    cls.language_code == language_code
+                )).order(sort)
+
+        sorted_results: List[
+            TranslationSubmitterTotalContributionStatsModel] = []
+        today = datetime.date.today()
+
+        if last_activity is not None:
+            last_date = today - datetime.timedelta(days=last_activity)
+            next_offset = offset
+            while len(sorted_results) < page_size:
+                result_models: Sequence[
+                    TranslationSubmitterTotalContributionStatsModel] = (
+                    sort_query.fetch(
+                        NUM_MODELS_PER_FETCH, offset=next_offset))
+                if not result_models:
+                    next_offset += 1
+                    break
+                for result_model in result_models:
+                    next_offset += 1
+                    if result_model.last_contribution_date >= last_date:
+                        sorted_results.append(result_model)
+                        if len(sorted_results) == page_size:
+                            break
+        else:
+            sorted_results = list(sort_query.fetch(page_size, offset=offset))
+            next_offset = offset + len(sorted_results)
+
+        # Check whether we have more results.
+        next_result_model: Sequence[
+            TranslationSubmitterTotalContributionStatsModel] = (
+                sort_query.fetch(offset=next_offset))
+        more: bool = len(next_result_model) != 0
+
+        return (
+            sorted_results,
+            next_offset,
+            more
+        )
+
+    @classmethod
+    def get_all_by_user_id(
+        cls, user_id: str
+    ) -> Sequence[TranslationSubmitterTotalContributionStatsModel]:
+        """Gets all TranslationSubmitterTotalContributionStatsModel matching
+        the supplied user_id.
+
+        Returns:
+            list(TranslationSubmitterTotalContributionStatsModel). The matching
+            TranslationSubmitterTotalContributionStatsModel.
+        """
+        return cls.get_all().filter(
+            cls.contributor_id == user_id
+        ).fetch(feconf.DEFAULT_SUGGESTION_QUERY_LIMIT)
+
+    @classmethod
+    def has_reference_to_user_id(cls, user_id: str) -> bool:
+        """Check whether TranslationSubmitterTotalContributionStatsModel
+        references the supplied user.
+
+        Args:
+            user_id: str. The ID of the user whose data should be checked.
+
+        Returns:
+            bool. Whether any models refer to the given user ID.
+        """
+        return cls.query(
+            cls.contributor_id == user_id
+        ).get(keys_only=True) is not None
+
+    @classmethod
+    def get_deletion_policy(cls) -> base_models.DELETION_POLICY:
+        """Model contains corresponding to a user: contributor_id."""
```

Comment:
This sentence doesn't scan.

Reply:
It was actually supposed to be "Model contains data corresponding to a user: contributor_id.". Updated